### DISCUSSION TITLE
OSMAND forces you to take a completely unnecessary diversion

### DISCUSSION THREAD

**Discussion by jobroukal** [osmandapp/OsmAnd#19360] [Maps Related]
My problem concerns the cycle route through Hasnerstraße in Vienna, Austria. In the direction to the city centre, a cycle path leads straight ahead across Schuhmeierplatz. OSMAND, however, forces you to take a completely unnecessary diversion and leave the route. The problem is probably that a mandatory diversion for vehicles is entered in OSM. However, with the exception "exceptions=bicycle". Graphhopper and other maps based on OSM therefore allow you to continue straight ahead. Please make this possible in OSMAND too!
![Screenshot 2024-03-19 175858](https://github.com/osmandapp/OsmAnd/assets/32236499/2973e0a6-e6d6-4c52-b697-ff48d0a1324c)

**Comment by vshcherb** [comment]
OsmAnd doesn't support resctrictions with except yet
